submitted_models/marble_spot_sensor_config_1/specifications.md

```diff
@@ -0,0 +1,90 @@
+# Boston Dynamics Spot - CTU-CRAS-Norlab sensor config
+This specifications.md file is a description and proof of virtual model validation for the
+Boston Dynamics Spot robot configured by team CTU-CRAS-Norlab. This robot may be launched using an
```

Comment:
Should CTU-CRAS-Norlab be Marble here?

Reply:
Fixed in https://github.com/osrf/subt/commit/7349e416ef7ccac9a4c3775ecf3057d6e17eef8d